src/db/queries.py

```diff
@@ -151,10 +152,22 @@ def load_all_events(latitude, longitude, start_date, end_date, number_plates):
             queries.append(CameraEvent.created_at <= end_date)   
         if number_plates:
             queries.append(CameraEvent.number_plate.in_(number_plates))
-        result = session.query(CameraEvent.latitude, CameraEvent.longitude, CameraEvent.number_plate)\
+        result = session.query(CameraEvent.id, CameraEvent.latitude, CameraEvent.longitude, CameraEvent.number_plate)\
                 .filter(or_(CameraEvent.longitude == longitude, CameraEvent.longitude == latitude))\
                 .filter(or_(CameraEvent.latitude == latitude, CameraEvent.latitude == longitude))\
                 .filter(*queries)\
                 .all()
 
-        return result
\ No newline at end of file
+        return result
+
+def load_event_trail_data(number_plate, date):

```

Comment:
Simon I know you dont like duck typing, and you can actually specify the type in python with something like this

```
def greeting(name: str) -> str:
    return 'Hello ' + name
```

for future ref :)

Reply:
That way the error will be throws if the type is wrong